docs/ssss.md

```diff
@@ -1,3 +1,101 @@
 # Simperby Super Simple Summary
 
-TODO
+## DAO
+
+A DAO (Decentralized Autonomous Organization) is an organization that operates
+autonomously on a blockchain without relying on government or centralized
+control. It is governed by a set of rules encoded in the protocol, and is based
+on a trustless network where all decisions are recorded and verified by the
+network participants.
+
+For more information on DAOs, refer to [the introduction to DAO
+(WIP)](./dao.md).
+
+## How to Implement a DAO
+
+The most common way to implement a DAO is by **using an existing mainnet that
+supports smart contracts**. This involves programming the governance rules of
+the DAO into a smart contract and deploying it on the mainnet. However, this
+approach **has some disadvantages**:
+
+1. Users need to pay **gas fees** to operate the DAO.
+2. It is reliant on the performance, security, reliability, usability, and
+   potential political risks **associated with the chosen mainnet**.
+3. Its capabilities are restricted by the ecosystem of the chosen mainnet and
+   may **not easily interact with other mainnets**.
```

Comment:
How about using the term "blockchain" instead of "mainnet"?

Reply:
fixed.